docs/configuration.md

```diff
@@ -1136,6 +1136,15 @@ combination with PAPERLESS_CONSUMER_BARCODE_UPSCALE bigger than 1.0.
 
     Defaults to "300"
 
+## Audit Trail
+
+#### [`PAPERLESS_AUDIT_ENABLED=<bool>`](#PAPERLESS_AUDIT_ENABLED){#PAPERLESS_AUDIT_ENABLED}
```

Comment:
Minor thing, obviously but I think should be something more like `PAPERLESS_AUDIT_LOG_ENABLED`